docs/capabilities/algorand-client.md

```diff
@@ -0,0 +1,38 @@
+# Algorand client
```

Comment:
This page needs more stuff, but this is a good starting point covering the stuff relevant for this pull request